src/pages/ExamplePage1/ExamplePage1.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function ExamplePage1() {
+  return (
+    <div className="App" style={{ width: '100vw', height: '100vw' }}>
```

Comment:
I think this was supposed to be `height: '100vh'` rather than `height: '100vw'`, but its nbd since these are example pages. I'm assuming every page needs to be wrapped in a div with `style={{ width: '100vw', height: '100vh' }`.